runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/DataflowWorkerHarnessHelper.java

```diff
@@ -57,18 +57,19 @@ public static DataflowWorkerHarnessOptions initializeGlobalStateAndPipelineOptio
 
     ExperimentContext ec = ExperimentContext.parseFrom(pipelineOptions);
 
-    String experimentName = Experiment.DisableConscryptSecurityProvider.getName();
-    if (!ec.isEnabled(Experiment.DisableConscryptSecurityProvider)) {
+    String experimentName = Experiment.EnableConscryptSecurityProvider.getName();
+    if (ec.isEnabled(Experiment.EnableConscryptSecurityProvider)) {
```

Comment:
In the previous PR which added this experiment I [mentioned](https://github.com/apache/beam/pull/7056#issuecomment-439556103) the need for performance testing if we were to change the default. Have you looked into this?

I can think of two scenarios to investigate:

1. Are there known workloads that needed this enabled for performance? For this, we could try to comb through the history from when it was originally added. Also we should monitor any benchmarks for regression after this is merged.
1. Could there be Beam/Dataflow customer workloads that need this enabled for performance? For this, we should make sure this change in default makes it into the release notes, with a pointer to this experiment for re-enabling it.

Reply:
1. We have a sense that IO bound pipelines could be affected up-to a 15%. (The number mentioned from the previous PR.) Likely typical regression will be much less.
2. This does not affect Beam in general. It affects Dataflow only. I think highlighting this in the release notes is a good idea. (Dataflow if they choose to do so could take additional proactive measures to communicate the change.)